skaffold.yaml

```diff
@@ -81,64 +93,45 @@ deploy:
 
 profiles:
   - name: dev
-    deploy:
-      helm:
-        releases:
-          - name: im-group-whoami-{{ .ENVIRONMENT }}
-            namespace: whoami
-            createNamespace: true
-            remoteChart: im-group
-            repo: https://dhis2-sre.github.io/im-group
-            version: 0.3.0
-            setValueTemplates:
-              serviceAccount:
-                name: im-manager-{{ .ENVIRONMENT }}
-                namespace: instance-manager-{{ .ENVIRONMENT }}
+    patches:
+      - op: add
+        path: /deploy/helm/releases/-
+        value:
+          name: im-group-whoami-{{ .ENVIRONMENT }}
+          namespace: whoami
+          <<: *chart-config
+          setValueTemplates:
+            serviceAccount:
+              name: im-manager-{{ .ENVIRONMENT }}
+              namespace: instance-manager-{{ .CLASSIFICATION }}
 
   - name: prod
-    deploy:
-      helm:
-        releases:
-          - name: im-group-dev
-            namespace: dev
-            createNamespace: true
-            remoteChart: im-group
-            repo: https://dhis2-sre.github.io/im-group
-            version: 0.3.0
-            setValues:
-              serviceAccount:
-                name: im-manager-prod
-                namespace: instance-manager-prod
+    patches:
+      - op: add
+        path: /deploy/helm/releases/-
+        value:
+          name: im-group-dev
+          namespace: dev
+          # TODO use YAML anchor
+          <<: [*chart-config, *service-account-config]
 
-          - name: im-group-play
```

Comment:
Did you remove `- name: im-group-play` and if so intentionally? It looks as if the same happened for `im-group-qa` and `im-group-meta-packages` 🤔

Reply:
The diff is slightly misleading here. Look at the added lines below or the actual contents of the file after the changes.